src/main/connector/system/System/System.yaml

```diff
@@ -126,4 +126,16 @@ metrics:
   system.uptime:
     description: System uptime in seconds indicates how long the host has been running since its last boot
     type: Gauge
-    unit: s
\ No newline at end of file
+    unit: s
+  system.volume.size.bytes:

```

Comment:
These are no longer required, as everything is now exposed with `system.filesystem` metrics, right?